docs/config-file-ort-yml.md

```diff
@@ -127,11 +128,9 @@ scopes defined in the examples below match the scopes in your project.
 
 ### When to Use Curations
 
-Curations should be used when you want to correct the licenses detected in the source code of the
-project. If you need to correct the license findings for a third-party dependency then add a curation to
-[curations.yml](config-file-curations-yml.md) or [package configuration](config-file-package-configuration-yml.md).
+License finding curations should be used when you want to correct the licenses detected in the source code of the project.
```

Comment:
This section is about both, package curations and license finding curations. Is it? ..but it only mentions the latter.

Reply:
Yes. I'll add the part about package configurations.

Reply:
I believe you mixed up "package curations" with "package configurations".